stats&plots/RFT_GLM_contrast.m

```diff
@@ -25,6 +25,7 @@
 %   - u: set-inducing threshold (for cluster inference)
 %   - verbose: flag for displaying results (default is 0)
 % OUT:
+%   - stat:
```

Comment:
There should be a description of what `stat`is.

Reply:
@jdaunize ?

Reply:
stat: vector containing the t- or F-values (depending on the test type selected in the input) corresponding to each point in Y